#Expyriment 1.0.0 (Python 3.12.2), .xpe-file, coding: UTF-8
#date: Mon Sep 29 2025 21:07:50
#sha1: 1a7eaf
#modules: 
#display: size=(800, 600), window_mode=False, opengl=2
#os: uname_result(system='Darwin', node='Riyas-MacBook-Air.local', release='25.0.0', version='Darwin Kernel Version 25.0.0: Mon Aug 25 21:12:01 PDT 2025; root:xnu-12377.1.9~3/RELEASE_ARM64_T8132', machine='x86_64')
Time,Type,Event,Value,Detail,Detail2
4585,stdout,received,'*** DEVELOP MODE ***'
4585,stderr,received,'Traceback (most recent call last):'
4585,stderr,received,'  File "/Users/riyajain/Desktop/Fall 2025/CORE-1/rectangle-kanisza.py", line 43, in <module>'
4586,stderr,received,'    control.start(exp,auto_create_subject_id=True)'
4586,stderr,received,"TypeError: start() got multiple values for argument 'auto_create_subject_id'"
